app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -0,0 +1,142 @@
+$image-overlay-form-width: 350px;
+
+.alchemy-image-overlay {
+
+  &.open {
+    background-color: rgba(0, 0, 0, 0.6);
+  }
+}
+
+.alchemy-image-overlay-container,
+.alchemy-image-overlay-dialog {
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.alchemy-image-overlay-body {
+  width: 100%;
+  height: 100%;
+}
+
+.alchemy-image-overlay-close {
+  width: 32px;
+  height: 32px;
+  top: 2*$default-padding;
+  right: $image-overlay-form-width + 2*$default-padding;
+  cursor: pointer;
+
+  .icon.close.small {
+    font-size: 32px;
+    color: white;
+  }
+}
+
+.picture-details-overlay {
+  @include box-sizing(border-box);
+  position: absolute;
```

Comment:
Properties should be ordered background-color, height, overflow, padding, position, right, top, width